src/components/Profile/ProfileInfo/ProfileData/Contact.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import style from '../ProfileInfo.module.css';
```

Comment:
It's better to move the related styles in the contact.module.css - so that this component will be better decoupled from the rest of the ProfileInfo.

Reply:
yep